public/static/docs/command-reference/diff.md

```diff
@@ -46,148 +57,142 @@ for example when `dvc init` was used with the `--no-scm` option.
 
 ## Examples
 
-For these examples we can use the chapters in our
-[Get Started](/doc/get-started) section, up to
-[Add Files](/doc/get-started/add-files).
+For these examples we can use the [Get Started](/doc/get-started) repository.
```

Comment:
Good simplification. I would go with
```suggestion
For these examples we can use the [Get Started](/doc/get-started) project.
```

p.s. this can probably be changed in several other docs as well.